app/services/google/search_service.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module Google
+  class SearchService
+    SearchServiceError = GoogleScraperRuby::Errors::SearchServiceError
+    HOST = 'www.google.com'
+    # rubocop:disable Layout/LineLength
+    USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36'
+    # rubocop:enable Layout/LineLength
+
+    def self.search!(keyword)
+      uri = URI::HTTPS.build(host: HOST, path: '/search', query: { q: keyword }.to_query)
+      conn = http_client uri
+      response = conn.get(uri, {}, { 'User-Agent' => USER_AGENT })
```

Comment:
Code style improvemens: add blank lines, rename variables to complete names and more consistency in using parentheses (you used them for `URI::HTTPS.build` but not for the line below)

```suggestion
      uri = URI::HTTPS.build(host: HOST, path: '/search', query: { q: keyword }.to_query)
      client = http_client(uri)
      response = client.get(uri, {}, { 'User-Agent' => USER_AGENT })
      
```